package.json

```diff
@@ -4,12 +4,12 @@
     "build": "yarn run build-js && yarn run build-css",
     "build-css": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' && postcss --use cssnano --dir static/minified 'static/css/**/*.css'",
     "build-js": "cp node_modules/d3/build/d3.min.js static/js/modules && cp node_modules/d3-geo/build/d3-geo.min.js static/js/modules && cp node_modules/topojson-client/dist/topojson-client.min.js static/js/modules",
-    "watch": "watch -p 'static/sass/*.scss' -c 'yarn run build-css'",
+    "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
```

Comment:
This will make it build Javascript every time sass files change

Reply:
I just noticed that